### PR TITLE
add tests with entityNameExp with id number

### PR DESCRIPTION
**Pull request by AlvaroVega**
just a test for https://github.com/telefonicaid/iotagent-node-lib/pull/1480 https://github.com/telefonicaid/iotagent-node-lib/pull/1487

### CODE REVIEW

File: test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js [hunk 1]
```diff
@@ -141,6 +141,26 @@ const iotAgentConfig = {
...
+            commands: [],
+            type: 'WeatherStation',
+            entityNameExp: 'id',
+            lazy: [],
+            /* info which will be part of typeInformation passed to iotalib.update*/
+            service: 'smartgondor',
+            subservice: 'gardens',
+            name: '1234',
+            id: '1234',
+            /* end info */
```
Comment (mapedraza) [inline test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js:154]: is this required?

Comment (AlvaroVega) [reply test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js:154]: Really service, subservice, id and type are expected in typeInformation received by iotagent.update function

Comment (AlvaroVega) [reply test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js]: Test updated in https://github.com/telefonicaid/iotagent-node-lib/pull/1484/commits/34fea35a65c215d6b5f58d2059070c86ba7d42b3

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**